CHANGELOG.md

```diff
@@ -1,11 +1,30 @@
 # Release History
 
+## 1.4.0 (TBD)
+
+### New Features
+- Added support for `session.getOrCreate` and `session.appName`
+- Added support for `functions.array_distinct`
+- Added support for alias `Column.getField`
+- Added support for `functions.regexp_extract`
+- Added support for `functions.struct`
+- Added support for `functions.daydiff` which is compatible with spark datediff
+- Added support for `functions.date_add` and `functions.date_sub` to ease common day add/substract operations
+- Added support for `functions.format_number`
+- Added support for `functions.bround`
+
+### Bug Fixes
+
+- Fixed a bug with `strtok_to_array` when a delimiter was passed it was throwing an exception
+
```

Comment:
nit: this is also present in bug fixes section in 1.3.0

Reply:
I moved this the new release, as this was found and added during my PR